integration/types/datatype_test.rb

```diff
@@ -139,8 +145,10 @@ def test_can_insert_each_primitive_datatype
     ensure
       cluster.close
     end
+  end
 
-    def test_all_collection_types_initially_nil
+  def test_all_collection_types_initially_nil
+    begin
```

Comment:
A method definition is an implicit begin/end block, so none of your methods in this file need to start with an explicit begin. You can do something like this:

```
def foo
   some_logic
ensure
   whatever
end
```